doc/manuals/poncho/index.md

```diff
@@ -57,14 +63,11 @@ Then to create a complete package from the specification:
 poncho_package_create package.json
 ```
 
-The outputed tarball will be put into a directory named `envs` if a specific 
-cache directory is not specified.
-
 Once created, this package can be moved to another machine for execution.
 Then, to run a program in the environment:
 
 ```
-poncho_package_run -e package.tar.gz -- example.py
+poncho_package_run -e package.tar.gz python example.py
```

Comment:
Let's keep the -- here, e.g.: `poncho_package_run -e package.tar.gz -- python example.py`